common/trajectory.cpp

```diff
@@ -90,6 +91,34 @@ nav_msgs::msg::Path Trajectory::toPath() const
 	return path;
 }
 
+foxglove_msgs::msg::GeoJSON Trajectory::toGeoJSON(double llh_0[3]) const {
```

Comment:
We are writing C++ code here :) std::vector saves you the trouble of passing and checking the size of the array as well as the pointer validity, which you would otherwise need to do.
```suggestion
foxglove_msgs::msg::GeoJSON Trajectory::toGeoJSON(const std::vector<double>& llh_0) const {
```